sdcm/tester.py

```diff
@@ -932,11 +985,11 @@ def get_cluster_k8s_gce_minikube(self):
         self.k8s_cluster.docker_pull(f"{self.params.get('docker_image')}:{self.params.get('scylla_version')}")
 
         self.db_cluster = \
-            minikube.MinikubeScyllaPodCluster(k8s_cluster=self.k8s_cluster,
-                                              scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
-                                              user_prefix=self.params.get("user_prefix"),
-                                              n_nodes=self.params.get("n_db_nodes"),
-                                              params=self.params)
+            minikube.RemoteMinimalScyllaPodCluster(k8s_cluster=self.k8s_cluster,
+                                                   scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
+                                                   user_prefix=self.params.get("user_prefix"),
+                                                   n_nodes=self.params.get("n_db_nodes"),
+                                                   params=self.params)
```

Comment:
Can you please write it in the following way? - 
```
self.db_cluster = minikube.RemoteMinimalScyllaPodCluster(
    k8s_cluster=self.k8s_cluster,
    scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
    user_prefix=self.params.get("user_prefix"),
    n_nodes=self.params.get("n_db_nodes"),
    params=self.params,
)
```

Reply:
fixed